ks/azure/centos76-hpc-sriov.ks

```diff
@@ -253,7 +253,7 @@ EOF
 
 
 cd /tmp
-CENTOS_HPC_VERSION="centos-hpc-20200814"
+CENTOS_HPC_VERSION="centos-hpc-11052020"
```

Comment:
Why the inversion in version nomenclature? This will prevent the versions from sorting properly.

Reply:
thanks for catching this (y)

Reply:
Sure thing! 😁   Looks good now!  I'll merge it in!